package.json

```diff
@@ -33,6 +33,8 @@
     "@rhoas/app-services-ui-components": "1.11.8",
     "@rhoas/app-services-ui-shared": "0.15.4",
     "keycloak-js": "9.0.0",
+    "lodash.isequal": "^4.5.0",
```

Comment:
:+1: 
You made me realize I've used `lodash` elsewhere as a transitive dependency. I'll have to [fix it](https://issues.redhat.com/browse/MGDOBR-757).

Reply:
Good to know